surfer/tests/test_viz.py

```diff
@@ -37,6 +37,7 @@ def test_offscreen():
     """
     mlab.options.backend = 'auto'
     brain = Brain(*std_args, offscreen=True)
+    _ = brain.screenshot()
     shot = brain.screenshot()
     assert_array_equal(shot.shape, (800, 800, 3))
```

Comment:
It's probably a Mayavi quirk. Just add a comment here, too, about Mayavi giving the wrong resolution on OSX sometimes